source/_components/sensor.sense.markdown

```diff
@@ -51,7 +56,7 @@ password:
   type: string
 monitored_conditions:
   description: List of sensors to display in the front end.
-  required: true
+  required: false
```

Comment:
It his becomes optional, the option should be removed from the initial example on top of the page.
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.